db/models.py

```diff
@@ -29,19 +29,19 @@ class CircuitScheduleDoc(BaseModel):
     """EVC circuit schedule model"""
 
     id: str
-    date: Optional[str]
-    frequency: Optional[str]
-    interval: Optional[int]
+    date: Optional[str] = None
+    frequency: Optional[str] = None
+    interval: Optional[int] = None
     action: str
 
 
 class TAGDoc(BaseModel):
     """TAG model"""
     tag_type: str
-    value: Union[int, str, list[list[int]]]
-    mask_list: Optional[list[str, int]]
+    value: Union[int, str, List[List[int]]]
```

Comment:
Looks like `list` should still work shouldn't it? Can you verify this again? Using the native/bultin types makes it even easier to type hints, check it out again.

Reply:
Changed